Description: fix FTBS on sparc64
  Upstream has fixed this differently in the 3.x branch (see upstream
  commit 97fa4ca15), this is a simpler patch that fixes our build until we
  package 3.x.
Author: David Matthew Mattli <dmm@mattli.us>
Forwarded: not-needed
Bug-Debian: https://bugs.debian.org/805072
--- a/src/google/protobuf/stubs/platform_macros.h
+++ b/src/google/protobuf/stubs/platform_macros.h
@@ -67,7 +67,7 @@
 #define GOOGLE_PROTOBUF_ARCH_32_BIT 1
 #elif defined(sparc)
 #define GOOGLE_PROTOBUF_ARCH_SPARC 1
-#ifdef SOLARIS_64BIT_ENABLED
+#if defined(SOLARIS_64BIT_ENABLED) || defined(__LP64__)
 #define GOOGLE_PROTOBUF_ARCH_64_BIT 1
 #else
 #define GOOGLE_PROTOBUF_ARCH_32_BIT 1
